docs/posts.md

```diff
@@ -0,0 +1,184 @@
+      "Yo"
+    ],
+    "channels": [
+      "test"
+    ],
+    "postType": "REPOST",
+    "reference": "zb2rhbnwaLFrTfw8uGzZKPfS16SLt14FmEFNghrGRZUhY3g2Y",
+    "timestamp": "2018-10-11T12:08:38.639363790Z"
+  },
+  "hash": "zb2rheDrgxGWWfiJBnFCAp7xwmK2Yt1AistCBzaYCrFB1adpN",
+  "signature": "NVg62CekPbI+3j6YvaTokSi0H8Z5cbMgTW84yT+P5U6E1mYg/0vdF3c6OggW9BcSd9bhQR6wfgaWgurxmi3OBQ=="
+}
+```
+
+### `POST /ob/post`
+
+This API is used to create a post, and requires the following payload:
+
+```JSON
+{
+	"status": "testy7",
+	"longForm": "This is a test post dawg.",
+	"postType": "POST",
+	"reference": "",
+	"images": [
+		{
+			"filename": "cat",
+			"large": "zb2rhmBUB9i7UkfmeD3obJYK3FFS5K8N8QHaUanG8UWLVBHiY",
+			"medium": "zb2rhaFhqziCWk1zo5tMRxQEUchfvJFaGG4DY1anEoR4GnYrN",
+			"original": "zb2rhe2o6WbHqcER5VUKsMUbQrmpCC6ihg8qZ4JS9wVgKz9wm",
+			"small": "zb2rhbDCeEiTTunugWPaRRKFCfNKUaB7aCR53nrPnMa9usZXY",
+			"tiny": "zb2rhgqJDbshwAgPjs7X2h4mDm3V3BpLbp4tFGqkg1LNkg9yV"
+		}
+	],
+	"tags": [
+		"Yo"
+	],
+	"channels": [
+		"test"
+	]
+}
+```
+
+### `PUT /ob/post`
+
+This API is used to edit an existing post. The payload must use the `slug` of the intended post to be edited:
+
+```JSON
+{
+	"status": "testy7",
+	"longForm": "This is a test post son.",
+	"postType": "POST",
+	"reference": "",
+	"images": [
+		{
+			"filename": "cat",
+			"large": "zb2rhmBUB9i7UkfmeD3obJYK3FFS5K8N8QHaUanG8UWLVBHiY",
+			"medium": "zb2rhaFhqziCWk1zo5tMRxQEUchfvJFaGG4DY1anEoR4GnYrN",
+			"original": "zb2rhe2o6WbHqcER5VUKsMUbQrmpCC6ihg8qZ4JS9wVgKz9wm",
+			"small": "zb2rhbDCeEiTTunugWPaRRKFCfNKUaB7aCR53nrPnMa9usZXY",
+			"tiny": "zb2rhgqJDbshwAgPjs7X2h4mDm3V3BpLbp4tFGqkg1LNkg9yV"
+		}
+	],
+	"tags": [
+		"Yo"
+	],
+	"channels": [
+		"test"
+	]
+}
+```
+
+### `DELETE /ob/post`
+
+This API is used to delete a post. The `slug` of the target post must be included in the URL path: `DELETE /ob/post/{slug}`.
+
+## Future work
+
+There are two primary areas of work that need improvments:
+
+1. Feed
+2. Reactions
+
+### Feed
+
+The 'feed' is simply a list of posts based on the nodes you follow, or posts published to a certain channel or with a tag. The feed will not require a user to fetch a list of posts manually from each node that they follow. Currently, third parties can construct a social feed of posts based on a list of `peerIds`, a channel, or tag. We aim to explore technologies such as IPFS pubsub and [OrbitDB](https://github.com/orbitdb/orbit-db) in further detail to remove this requirement and allow a more decentralized construction of feeds.
+
+### Reactions
+
+'Reactions' are social reactions: like, love, smile etc. While this data could exist as its own type of post, this solution isn't scalable and would require frequent republishing of the root IPNS directory, and undermining the ability of peers to seed the latest version of the store. A decentralised solution is required to handle rapidly dynamic data; again, OrbitDB seems like a good candidate to investigate.
```

Comment:
Let's remove this doc from here (as it relates primarily to setup related documentation). The API details can go into Postman as described. 

The rest of the content should either update (the existing OBIP)[https://github.com/OpenBazaar/obips/blob/master/obip-0004.md] or in an amendment OBIP. What do you recommend for that, @tyler-smith?

Reply:
@hoffmabc expressed some desire for this to be documented, so I'd like to get him to weigh in on this.

Reply:
I'm not arguing for NO documentation. I'm arguing that move what you wrote to an appropriate place. The API calls and responses should be in Postman with the rest of the API documentation. The rest of the features can be described in the OBIP with the rest of the explanation for how posts work. No docs should be lost here from this.

Reply:
What did we agree on at the retreat regarding docs in the code and API docs? Let’s not deviate from what we agreed upon during the cleanup session.